

drivers/input/mouse/psmouse-base.c: In function `psmouse_do_set_rate':
drivers/input/mouse/psmouse-base.c:55: warning: passing arg 4 of `psmouse_attr_set_helper' from incompatible pointer type
drivers/input/mouse/psmouse-base.c: In function `psmouse_do_set_resolution':
drivers/input/mouse/psmouse-base.c:56: warning: passing arg 4 of `psmouse_attr_set_helper' from incompatible pointer type
drivers/input/mouse/psmouse-base.c: In function `psmouse_do_set_resetafter':
drivers/input/mouse/psmouse-base.c:57: warning: passing arg 4 of `psmouse_attr_set_helper' from incompatible pointer type
drivers/input/mouse/psmouse-base.c: At top level:
drivers/input/mouse/psmouse-base.c:857: conflicting types for `psmouse_attr_set_helper'
drivers/input/mouse/psmouse.h:89: previous declaration of `psmouse_attr_set_helper'

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-power4-akpm/drivers/input/mouse/psmouse.h |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/input/mouse/psmouse.h~psmouse-build-fix drivers/input/mouse/psmouse.h
--- 25-power4/drivers/input/mouse/psmouse.h~psmouse-build-fix	2004-10-01 00:59:23.100001608 -0700
+++ 25-power4-akpm/drivers/input/mouse/psmouse.h	2004-10-01 00:59:29.086091584 -0700
@@ -86,7 +86,7 @@ void psmouse_set_resolution(struct psmou
 ssize_t psmouse_attr_show_helper(struct device *dev, char *buf,
 			ssize_t (*handler)(struct psmouse *, char *));
 ssize_t psmouse_attr_set_helper(struct device *dev, const char *buf, size_t count,
-			int (*handler)(struct psmouse *, const char *, size_t));
+			ssize_t (*handler)(struct psmouse *, const char *, size_t));
 
 #define PSMOUSE_DEFINE_ATTR(_name)						\
 static ssize_t psmouse_attr_show_##_name(struct psmouse *, char *);		\
_
